numpy/core/tests/test_multiarray.py

```diff
@@ -3708,7 +3708,7 @@ def test_highest_available_pickle_protocol(self):
         except ImportError:
             pickle5 = None
 
-        if sys.version_info[:2] >= (3, 8) or pickle5 is not None:
+        if sys.version_info[:4] >= (3, 8, 0, "alpha") or pickle5 is not None:
```

Comment:
Assuming alpha release will have it for now (in which case the change is not too important, but seems safe). I am good with waiting until then (or maybe at least until the 1.17 release).